### PR TITLE
threat-dragon, chainloop, airflow, kestra, gitleaks RT env

### PR DESCRIPTION
**Pull request by sand-hya**
threat-dragon: https://github.com/Be-Secure/Be-Secure/issues/185
chainloop: https://github.com/Be-Secure/Be-Secure/issues/521
airflow: https://github.com/Be-Secure/Be-Secure/issues/540
kestra: https://github.com/Be-Secure/Be-Secure/issues/504
gitleaks: https://github.com/Be-Secure/Be-Secure/issues/340

### CODE REVIEW

File: airflow/0.0.1/besman-airflow-RT-env.sh [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        sudo apt install python3 -y
+    else
+        echo "Python is already there to use."
+    fi
+
+    # Check if pip is installed
+    if ! command -v pip3 &>/dev/null; then
+        echo "pip is not installed. Installing pip..."
+        sudo apt update
+        sudo apt install python3-pip -y
```
Comment (asa1997) [inline airflow/0.0.1/besman-airflow-RT-env.sh:53]: @sand-hya dont we have an ansible role for python?

File: airflow/0.0.1/besman-airflow-RT-env-config.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+BESMAN_ASSESSMENT_DATASTORE_URL: https://github.com/Be-Secure/besecure-assessment-datastore #***
+
+# The path where we download the ansible role of the assessment tools and other utilities
+BESMAN_ANSIBLE_ROLES_PATH: $BESMAN_DIR/tmp/$BESMAN_ARTIFACT_NAME/roles #***
+
+# The list of tools you wish to install. The tools are installed using ansible roles.
+# To get the list of ansible roles run 
+#   $ bes list --role
+#add the roles here. format - <Github id>/<repo name>,<Github id>/<repo name>,<Github id>/<repo name>,... #***
+BESMAN_ANSIBLE_ROLES: Be-Secure/ansible-role-oah-criticality_score,Be-Secure/ansible-role-oah-docker,Be-Secure/ansible-role-oah-sonarQube,Be-Secure/ansible-role-oah-vscode
```
Comment (asa1997) [inline airflow/0.0.1/besman-airflow-RT-env-config.yaml:53]: Please validate the working of vscode ansible role.

**Review by asa1997 — COMMENTED** [review]
@sand-hya Please address the comments.

**Comment by sand-hya** [comment]
raising new PRs for each projects separately

**Pull request closed without merging**